.ci/magic-modules/generate-terraform-all-platforms.yml

```diff
@@ -0,0 +1,19 @@
+---
+platform: linux
+inputs:
+  - name: terraform-generated
+  - name: magic-modules
+
+image_resource:
+  type: docker-image
+  source:
+    repository: golang
+    tag: '1.9.3'
```

Comment:
Recommend you update to 1.10 if possible, also recommend using minor version specifier instead of patch version.

Reply:
Done